### DISCUSSION TITLE
Oxygen Theme!

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2456] [Show and tell]
This has been a long, long time coming. It is by far the most difficult theme I have ever made, but I learned a lot about SVG images and the radial gradient primitives in QSS at least. Source and screenshots <a href=https://github.com/AzumaHazuki/lxqt-themes-oxygen>available here</a>.

I'd like to think this is done, but any theme this complex is bound to have issues. If anyone notices anything seriously off, please bring it to my attention. To me, this is a kind of preservation project, as well as a way of bringing advanced theming to low-resource computers that might have some issues with a full-fat KDE desktop.

![LXQt-Oxygen](https://github.com/lxqt/lxqt/assets/67122280/35277a8a-884e-4c34-80ff-6f4840a6bdde)

**Comment by Chiitoo** [comment]
Yes and also please.

Thank you!
